packages/server/modules/core/services/objects.js

```diff
@@ -1,84 +1,18 @@
 const { performance } = require('perf_hooks')
-const { set, get, chunk } = require('lodash')
+const { set, get } = require('lodash')
 
 const knex = require(`@/db/knex`)
 const { servicesLogger } = require('@/logging/logging')
 const { chunkInsertionObjectArray } = require('@/modules/core/utils/chunking')
-const { prepInsertionObject } = require('@/modules/core/services/objects/management')
+const {
+  prepInsertionObject,
+  prepInsertionObjectBatch
+} = require('@/modules/core/services/objects/management')
 
 const Objects = () => knex('objects')
 const Closures = () => knex('object_children_closure')
 
 module.exports = {
-  async createObjectsBatched({ streamId, objects, logger = servicesLogger }) {
-    const closures = []
-    const objsToInsert = []
-    const ids = []
-
-    // Prep objects up
-    objects.forEach((obj) => {
-      const insertionObject = prepInsertionObject(streamId, obj)
-      let totalChildrenCountGlobal = 0
-      const totalChildrenCountByDepth = {}
-
-      if (obj.__closure !== null) {
-        for (const prop in obj.__closure) {
-          closures.push({
-            streamId,
-            parent: insertionObject.id,
-            child: prop,
-            minDepth: obj.__closure[prop]
-          })
-          totalChildrenCountGlobal++
-          if (totalChildrenCountByDepth[obj.__closure[prop].toString()])
-            totalChildrenCountByDepth[obj.__closure[prop].toString()]++
-          else totalChildrenCountByDepth[obj.__closure[prop].toString()] = 1
-        }
-      }
-
-      insertionObject.totalChildrenCount = totalChildrenCountGlobal
-      insertionObject.totalChildrenCountByDepth = JSON.stringify(
-        totalChildrenCountByDepth
-      )
-
-      delete insertionObject.__tree
-      delete insertionObject.__closure
```

Comment:
dropped these calls, same reason as in the other PR (there doesn't appear to be a way these could exist)